app/routes/wow.ilvl-export-search.tsx

```
@@ -0,0 +1,506 @@
+}
+
+type LoaderResponseType =
+  | {}
+  | { exception: string }
+  | (IlvlExportResponse & {
+      sortby: string
+      formValues: typeof defaultFormValues
+    })
+
+type ActionResponseType =
+  | {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)